gammapy_mwl/io/ogip.py

```diff
@@ -39,15 +39,21 @@ def from_hdulist(cls, hdulist, hdu1="MATRIX", hdu2="EBOUNDS"):
 
     pdf_matrix = np.zeros([len(data), header["DETCHANS"]], dtype=np.float64)
 
+    #check for TLMIN keyword to determine if indexing starts at 0 or 1:
+    try:
```

Comment:
OGIP says TLMIN<N> where N is the column number of the `F_CHAN` column.

So I'd change to:

```
col_num = matrix_hdu.columns.names.index("F_CHAN") + 1
matrix_hdu.header.get(f"TLMIN{col_num}", 0)
```